### ISSUE TITLE
Upgrade to Wagtail 2.8.1

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#308]
## Need
In order to make security patches and because we are long overdue, we need to upgrade to the newest version of Wagtail which is now `2.8.1`. This should be done in a stepped up fashion, meaning we upgrade each major version of Django (as needed) and Wagtail one at a time and resolve issues with the code for each version before moving on to the next. This should not be done before Rumsey's training on Wednesday April 22nd.

We are currently using `Django 2.2.10` and `Wagtail 2.5.1`.

### TODO
- [x] Read the Django and Wagtail release notes for every version between our current version and the next.
- [x] Implement security patches
- [ ] Write some form of documentation for a loop news post

#### Nice to have
- [x] Upgrade to `Django 3.0.5` 

### Dependencies
The following issues should be resolved before we start on this. These are blockers:
- [x] #306 
- [x] #307
